DigitalOffice/Services/ProjectService/Validators/EditProjectValidator.cs

```diff
@@ -0,0 +1,35 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.ProjectService.Models;
+using System;
+
+namespace LT.DigitalOffice.ProjectService.Validators
+{
+    public class EditProjectValidator : AbstractValidator<EditProjectRequest>
+    {
+        public EditProjectValidator()
+        {
+            RuleFor(project => project.Id)
+                .NotEmpty()
+                .WithMessage("Request must have a project Id");
+
+            RuleFor(project => project.Name)
+                    .NotEmpty()
+                    .WithMessage("Project must have a name.")
+                    .MaximumLength(80)
+                    .WithMessage("Project name is too long.");
+
+            RuleFor(project => project.Description)
+                .MaximumLength(500)
+                .WithMessage("Project description is too long.");
+        }
+
+        private bool NotGuidEmpty(Guid guid)
```

Comment:
Why this code?

Reply:
Obviously I've just forgotten this. You can see this by private modifier and that it is not used anywhere in the class, no need to address me the "why" - you can clearly tell by yourself

Reply:
removed